File: gto-support-appcompat/src/main/java/androidx/appcompat/widget/AppCompatTextViewImpl.java
package androidx.appcompat.widget;

import android.content.Context;
import android.util.AttributeSet;

import androidx.annotation.NonNull;

import static androidx.appcompat.widget.ReflectionUtils.getDeclaredField;

public class AppCompatTextViewImpl extends AppCompatTextView {
    @NonNull
    private final AppCompatCompoundDrawableHelper mCompoundDrawableHelper;

    public AppCompatTextViewImpl(Context context) {
        this(context, null);
    }

    public AppCompatTextViewImpl(Context context, AttributeSet attrs) {
        this(context, attrs, android.R.attr.textViewStyle);
    }

    public AppCompatTextViewImpl(Context context, AttributeSet attrs, int defStyleAttr) {
        super(context, attrs, defStyleAttr);

        mCompoundDrawableHelper =
                new AppCompatCompoundDrawableHelper(this, getDeclaredField(AppCompatTextView.class, "mTextHelper"));
        mCompoundDrawableHelper.loadFromAttributes(attrs, defStyleAttr);
    }

    @Override
    protected void drawableStateChanged() {
        super.drawableStateChanged();
        mCompoundDrawableHelper.applyCompoundDrawablesTints();
    }
}

File: gto-support-appcompat/src/main/java/androidx/appcompat/widget/AppCompatTextViewImpl.kt
package androidx.appcompat.widget

import android.content.Context
import android.util.AttributeSet

@Deprecated("Since v3.3.0, use AppCompatTextView directly")
class AppCompatTextViewImpl @JvmOverloads constructor(
    context: Context,
    attrs: AttributeSet? = null,
    defStyleAttr: Int = android.R.attr.textViewStyle
) : AppCompatTextView(context, attrs, defStyleAttr)
